projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -611,26 +613,75 @@ function wpcom_add_scheduled_updates_menu() {
 add_action( 'admin_menu', 'wpcom_add_scheduled_updates_menu' );
 
 /**
- * Add the Plugins menu item to the admin menu on simple sites.
+ * Add the Plugins menu item to the admin menu
+ *
+ *  * Adds a Plugin Marketplace link to the Plugins menu in the admin sidebar.
+ *  * Adds Installed Plugins and Add New Plugin links to the Plugins menu in the admin sidebar if they don't exist
+ *  * Updates the capability of the Installed Plugins and Add New Plugin links to manage_options if they do exist
  */
 function wpcom_add_plugins_menu() {
-
-	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+	if ( ! function_exists( 'wpcom_is_nav_redesign_enabled' ) || ! wpcom_is_nav_redesign_enabled() ) {
 		return;
 	}
+	global $menu;
+
+	$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+
+	$found_plugin_menu            = false;
+	$found_plugin_install_submenu = false;
+	foreach ( $menu as &$menu_item ) {
+		if ( 'plugins.php' === $menu_item[2] ) {
+			$found_plugin_menu = true;
```

Comment:
I'm failing to see under which circumstances this menu can exist for Simple sites, so I'm considering to simplify the logic here (and have something similar to what we had before).